### ISSUE TITLE
Unable to edit a policy after creation

### ISSUE THREAD

**Issue by chihaiaalex** [F5Networks/terraform-provider-bigip#384]
### Environment

 * TMOS/Bigip Version: BIG-IP 12.1.2 Build 2.0.276 Hotfix HF2
 * Terraform Version: v0.13.5
 * Terraform bigip provider Version: 1.4.0

### Summary

I am unable to edit a policy after creation. I tried multiple combinations by also providing the `published_copy` parameter, but with no success.

### Steps To Reproduce

```
resource "bigip_ltm_pool" "test_pool" {
  name = "/k8s/pool_TCP31380_POOL"
  load_balancing_mode = "round-robin"
  description = "Test pool"
  monitors = [
    "/Common/tcp"
  ]
  allow_snat = "yes"
  allow_nat = "yes"
}

resource "bigip_ltm_policy" "test_policy" {
  name = "/k8s/test-policy"
  strategy = "first-match"
  requires = ["http"]
  controls = ["forwarding", "persistence"]

  rule {
    name = "test-rule"

    condition {
      http_host = true
      host = true
      equals = true
      values = [
        "hello.example.com"
      ]
    }

    action {
      forward = true
      pool = bigip_ltm_pool.test_pool.name
    }

    action {
      disable = true
      persist = true
    }
  }
}
```
When I run `terraform apply` for the first time, everything is ok, the policy is created and published:
```
Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions in workspace "test"?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.bigip.bigip_ltm_policy.test_policy: Creating...
module.bigip.bigip_ltm_policy.test_policy: Creation complete after 1s [id=/k8s/test-policy]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

When I try to edit it, changing for example the values list for a rule:

```
# module.bigip.bigip_ltm_policy.test_policy will be updated in-place
  ~ resource "bigip_ltm_policy" "test_policy" {
  ...
 ~ rule {
            name = "test-rule"
  ...
    ~ values = [
           -  "hello.example.com",
           + "hello-changed.example.com",
    ]
 ...
Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions in workspace "test"?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.bigip.bigip_ltm_policy.test_policy: Modifying... [id=/k8s/test-policy]

Error: 01020036:3: The requested Policy (/k8s/Drafts/k8s/test-policy) was not found.

  on modules\bigip\main.tf line 329, in resource "bigip_ltm_policy" "test_policy":
 329: resource "bigip_ltm_policy" "test_policy" {
```

### Expected Behavior

I would expect the policy to be edited successfully and published.

### Actual Behavior

The policy is not edited and I get an error.

**Comment by chihaiaalex** [comment]
I've opened up a pull request for this issue: #387

**Comment by papineni87** [comment]
Fixed in v1.6.0
